### PR TITLE
Add ability to redraw immediately

### PR DESCRIPTION
**Pull request by AydinHassan**
This one adds the ability to immediately redraw the menu. I've also started off the tests for the menu, it's pretty easy and we can probably get 100% coverage now.

### CODE REVIEW

**Comment by AydinHassan** [comment]
@mikeymike next one :D

File: src/CliMenu.php [hunk 1]
```diff
@@ -235,6 +235,18 @@ protected function executeCurrentItem()
     }
 
     /**
+     * Redraw the menu
+     */
+    public function reDraw()
```
Comment (mikeymike) [inline src/CliMenu.php]: sorry to be pedantic but redraw is one word right? it just looks a little odd as `reDraw` what do you think ?

Comment (AydinHassan) [inline src/CliMenu.php]: yeah I agree, it did look a little weird

Comment (AydinHassan) [inline src/CliMenu.php]: @mikeymike fixed

**Pull request merged**